x-pack/plugins/lens/public/editor_frame_plugin/plugin.tsx

```diff
@@ -29,13 +30,19 @@ export class EditorFramePlugin {
       mount: element => {
         unmount();
         domElement = element;
+
+        const firstDatasourceId = Object.keys(this.datasources)[0];
+        const firstVisualizationId = Object.keys(this.visualizations)[0];
```

Comment:
Using key order might work, but I've been wondering about this in the registry- shouldn't we used something like an indexed map?

Reply:
This is just to pick something on first render - we should change how that works before first MVP (maybe a configuration setting with a meaningful default value or a "priority" prop on the datasources and visualizations?). Same thing for the dropdowns to switch between datasources and visualizations - just there to be able to test for now.

Not sure what an indexed map would help us with this - it's still kind of random which one gets picked because we won't control the loading order of plugins and picking the first one in an alphabetically sorted list also isn't really helpful.